allensdk/internal/brain_observatory/time_sync.py

```diff
@@ -339,8 +343,68 @@ def corrected_stim_timestamps(self):
         elif self.stim_data_length is None:
             logging.info("No data length provided for stim stream")
 
+        return timestamps, delta
+
+    @property
+    def clipped_stim_timestamps(self):
+        """
+        Return the stimulus timestamps with the erroneous initial spike
+        removed (if relevant)
+
+        Returns
+        -------
+        timestamps: np.ndarray
+            An array of stimulus timestamps in seconds with th emonitor delay
+            added
+
+        delta: int
+            Difference between the length of timestamps and the number of frames
+            reported in the stimulus pickle file, i.e.
+            len(timestamps) - len(pkl_file['items']['behavior']['intervalsms']
+        """
+        if self._clipped_stim_ts_delta is None:
+            (self._clipped_stim_timestamp_values,
+             self._clipped_stim_ts_delta) = self._get_clipped_stim_timestamps()
+
+        return (self._clipped_stim_timestamp_values,
+                self._clipped_stim_ts_delta)
+
+    def _get_monitor_delay(self):
```

Comment:
Would it be difficult/disruptive to rename the stand-alone function `monitor_delay()` (line 67) to something like `calculate_monitor_delay()` to distinguish from the class property?

Reply:
It is only disruptive if I misspell "calculate", which I did not my initial commit.

Done.